html-forms/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms!</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <div>
+        <label for="email"> Email </label><br />
+        <input type="email" value="codeguy@gmail.com" />
+      </div>
+      <div>
+        <label for="option">Why is coding cool?</label><br />
+        <select id="option">
+          <option value="Select">Select an option</option>
+          <option value="fun">Because it's fun</option>
+          <option value="Interesting">Because it's interesting</option>
+          <option value="'powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <div>
+        <label for="radio">Favorite Language(choose one)</label><br />
+
+        <input type="radio" id="HTML" />
+        <label for="HTML">HTML</label>
+
+        <input type="radio" id="CSS" />
+        <label for="CSS">CSS</label>
+
+        <input type="radio" id="JavaScript" />
+        <label for="JavaScript">JavaScript</label>
+      </div>
+      <div>
+        <label for="subscribe"
+          >Would you like to sing up for more information?</label
+        ><br />
```

Comment:
Since this `label` isn't associated with any field control element, this should just be a `p` element instead